servers/rendering/rasterizer_rd/shaders/tonemap.glsl

```diff
@@ -251,7 +251,9 @@ vec3 apply_bcs(vec3 color, vec3 bcs) {
 }
 
 vec3 apply_color_correction(vec3 color, sampler3D correction_tex) {
-	return texture(correction_tex, color).rgb;
+	ivec3 tex_size = textureSize(correction_tex, 0);
```

Comment:
I suggest if possible pass the texture size in the push constant, because textureSize is akin to a texture read.